doc/classes/OS.xml

```diff
@@ -243,8 +243,8 @@
 		<method name="get_granted_permissions" qualifiers="const">
 			<return type="PackedStringArray" />
 			<description>
-				With this function, you can get the list of dangerous permissions that have been granted to the Android application.
-				[b]Note:[/b] This method is implemented on Android.
+				On Android devices: with this function, you can get the list of dangerous permissions that have been granted.
+				On macOS (sandboxed applications only): this function returns the list of user selected folders accessible to the application. Use native file dialog to request folder access permission.
```

Comment:
```suggestion
				On Android devices: With this function, you can get the list of dangerous permissions that have been granted.
				On macOS (sandboxed applications only): This function returns the list of user selected folders accessible to the application. Use native file dialog to request folder access permission.
```
That being said, it feels like a pretty specific use case for reusing this generic method.